app/models/claim.rb

```diff
@@ -23,10 +23,17 @@ class Claim < ApplicationRecord
   belongs_to :school
   belongs_to :provider
 
-  has_many :mentor_trainings
-  has_many :mentors, through: :mentor_trainings
+  has_many(
+    :mentor_trainings,
```

Comment:
Does this need to be `dependent: :destroy`?

Reply:
Yeah, though we probably won't destroy a claim but I'll add it for completion

Reply:
I don't think this was done.
But @ollietreend made a point on my PR that we can protect with `restrict_with_exception` instead https://github.com/DFE-Digital/itt-mentor-services/pull/334#discussion_r1507718744

Reply:
Yeah I made the same point here https://github.com/DFE-Digital/itt-mentor-services/pull/332#discussion_r1507754121 I don't think we need `restrict_with_exception` as you cannot delete a claim if it has mentor trainings anyway

Reply:
You'll get something like this 
![Screenshot from 2024-02-29 15-53-31](https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/5c2acefd-d8f4-46de-98fb-b7cbf5a7d969)